src/main/java/roomescape/controller/RoomEscapeController.java

```diff
@@ -0,0 +1,49 @@
+package roomescape.controller;
+
+import java.util.List;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseBody;
+import roomescape.dto.request.ReservationRequest;
+import roomescape.dto.response.ReservationResponse;
+import roomescape.repository.ReservationRepository;
+
+@Controller
+public class RoomEscapeController {
+
+    private final ReservationRepository repository = new ReservationRepository();
+
+    @GetMapping("/admin")
+    public String adminPage() {
+        return "admin/index";
+    }
+
+    @GetMapping("/admin/reservation")
+    public String reservationAdminPage() {
+        return "admin/reservation-legacy";
+    }
+
+    @ResponseBody
+    @GetMapping("/reservations")
+    public List<ReservationResponse> reservationsJson() {
+        return ReservationResponse.reservationsToDtos(repository.findAll());
```

Comment:
repository에서 모든 예약들을 가져오는 로직과 변환 로직을 분리하면 좋겠어요.
```suggestion
        List<Reservations> reservations = repository.findAll();
        return ReservationResponse.reservationsToDtos(reservations);
```

Reply:
반영했습니다! 중간 변수를 제거하는 것이 더 간결하다고 생각했는데, 코드의 명확성과 이해를 위해서는 분리하는 것이 더 유리하다고 생각합니다.

[refactor(controller): findAll() 호출 결과를 변수로 추출하여 가독성 향상](https://github.com/woowacourse/spring-roomescape-admin/pull/200/commits/27b47cb959402da599eaae513b2353fabe877507)